code/modules/mob/living/simple_animal/hostile/retaliate/clown.dm

```diff
@@ -44,3 +44,263 @@
+
+/mob/living/simple_animal/hostile/retaliate/clown/longface
+	name = "Longface"
+	desc = "Often found walking into the bar."
+	icon = 'icons/mob/clown_mobs.dmi'
+	icon_state = "long face"
+	icon_living = "long face"
+	move_resist = INFINITY
+	turns_per_move = 10
+	response_help = "tries awkwardly to hug"
+	response_disarm = "pushes the unwieldy frame of"
+	response_harm = "tries to shut up"
+	speak = list("YA-HONK!!!")
+	emote_see = list("honks", "squeaks")
+	speak_chance = 60
+	maxHealth = 150
+	health = 150
+	pixel_x = -16
+	speed = 10
+	harm_intent_damage = 5
+	melee_damage_lower = 5
+	attacktext = "YA-HONKs"
+	loot = list(/obj/item/clothing/mask/gas/clown_hat, /obj/effect/gibspawner/human, /obj/item/soap)
+
+/mob/living/simple_animal/hostile/retaliate/clown/clownhulk
+	name = "Honk Hulk"
+	desc = "A cruel and fearsome clown. Don't make him angry."
+	icon = 'icons/mob/clown_mobs.dmi'
+	icon_state = "honkhulk"
+	icon_living = "honkhulk"
+	move_resist = INFINITY
+	response_help = "tries desperately to appease"
+	response_disarm = "foolishly pushes"
+	response_harm = "angers"
+	access_card = ACCESS_THEATRE
+	speak = list("HONK", "Honk!", "HAUAUANK!!!", "GUUURRRRAAAHHH!!!")
+	emote_see = list("honks", "sweats", "grunts")
+	speak_chance = 5
+	maxHealth = 400
+	health = 200
+	pixel_x = -16
+	speed = 2
+	harm_intent_damage = 15
+	melee_damage_lower = 15
+	melee_damage_upper = 20
+	attacktext = "pummels"
+	obj_damage = 30
+	environment_smash = ENVIRONMENT_SMASH_WALLS
+	loot = list(/obj/item/clothing/mask/gas/clown_hat, /obj/effect/gibspawner/human, /obj/item/soap)
+
+/mob/living/simple_animal/hostile/retaliate/clown/clownhulk/chlown
+	name = "Chlown"
+	desc = "A real lunkhead who somehow gets all the girls."
+	icon_state = "chlown"
+	icon_living = "chlown"
+	response_help = "submits to"
+	response_disarm = "tries to assert dominance over"
+	response_harm = "makes a weak beta attack at"
+	speak = list("HONK", "Honk!", "Bruh", "cheeaaaahhh?")
+	emote_see = list("asserts his dominance", "emasculates everyone implicitly")
+	maxHealth = 500
+	health = 300
+	speed = -2
+	armour_penetration = 20
+	attacktext = "steals the girlfriend of"
+	attack_sound = 'sound/items/airhorn2.ogg'
+	loot = list(/obj/item/clothing/mask/gas/clown_hat, /obj/effect/gibspawner/human, /obj/effect/particle_effect/foam, /obj/item/soap)
+
+/mob/living/simple_animal/hostile/retaliate/clown/clownhulk/honcmunculus
+	var/datum/reagent/funjuice = "methamphetamine"
+	name = "Honkmunculus"
+	desc = "A slender wiry figure of alchemical origin."
+	icon_state = "honkmunculus"
+	icon_living = "honkmunculus"
+	response_help = "skeptically pokes"
+	response_disarm = "pushes the unwieldy frame of"
+	speak = list("honk")
+	emote_see = list("squirms", "writhes")
+	speak_chance = 1
+	maxHealth = 200
+	health = 100
+	speed = -5
+	harm_intent_damage = 5
+	melee_damage_lower = 5
+	melee_damage_upper = 10

```

Comment:
Maybe this kind of clown should also deal no direct damage on a strike, making it more of a "support clown"? It could still hurt/kill people by ODing them on and getting them addicted to meth, of course.